common/scala/src/main/scala/whisk/core/connector/Message.scala

```diff
@@ -53,7 +54,8 @@ case class ActivationMessage(
     activationId: ActivationId,
     activationNamespace: EntityPath,
     content: Option[JsObject],
-    cause: Option[ActivationId] = None)
+    cause: Option[ActivationId] = None,
+    traceMetadata: Option[SpanMetadata] = None)
```

Comment:
Thanks for explaining. I imagine we can handle the _shared state_ problem in a separate PR. Do we want to put this PR on hold in the meanwhile, or do we go with incremental changes, integrate this PR, then update the implementation once the _shared state_ is in place ?